src/registration/InterestInTerraCheckbox.tsx

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash/fp';
+import React from 'react';
+import { LabeledCheckbox } from 'src/components/common';
+
+interface InterestInTerraCheckboxProps {
+  title: string;
+  interestInTerra: string;
+  setFunc: (interest: string) => void;
```

Comment:
Suggest a different prop name to make it more clear what this is and to decouple it from its current consumers.

```suggestion
  onChange: (interest: string) => void;
```